main.go

```diff
@@ -181,3 +181,11 @@ func main() {
 	}
 	app.Run(os.Args)
 }
+
+func GetSanitizedUrl(url string) string {
+	if !strings.HasPrefix(url, "https://") && !strings.HasPrefix(url, "http://") {
+		url = "http://" + url
```

Comment:
This has an issue in that passing an empty string will return "http://". This causes the subsequent check for if server == "" to go through.

Reply:
Thanks! Good catch. Fixed that one and added a fitting test case for the GetSanitizedUrl() function.